client/components/DummyChild.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class Dummy extends React.Component {
+  render () {
+    return (
+      <div className='container-child'>
+        <h1> Im the child</h1>
```

Comment:
nit: I think this can be:

```suggestion
        <h1>I'm the child</h1>
```